cumulus/parachains/runtimes/constants/Cargo.toml

```diff
@@ -15,10 +15,12 @@ workspace = true
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
+hex-literal = { workspace = true, default-features = true }
```

Comment:
it looks like not used, right?
```suggestion
```

Reply:
fixed in https://github.com/paritytech/polkadot-sdk/pull/7402/commits/77b4ce702a609417877fa11602b17f33ef5770e5